src/data/businessCase.ts

```diff
@@ -42,3 +42,19 @@ export const businessCaseInitalData: BusinessCaseModel = {
   preferredSolution: defaultProposedSolution,
   alternativeA: defaultProposedSolution
 };
+
+export const prepareBusinessCaseForApi = (
```

Comment:
These two functions will change in #189. No big deal. This is just bare bones placeholder stuff until we have the backend ready.